### PR TITLE
Added limitations of RO volume from a volume snapshot

### PR DESCRIPTION
**Pull request by bpjain2004**
# Description
Added limitations of RO volume from a volume snapshot

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/troubleshooting/powerscale.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ Here are some installation failures that might be encountered and how to mitigat
...
\ No newline at end of file
+| The `kubectl logs isilon-controller-0 -n isilon -c driver` logs shows the driver **Authentication failed. Trying to re-authenticate** when using Session-based authentication | The issue has been resolved from OneFS 9.3 onwards, for OneFS versions prior to 9.3 for session-based authentication either smart connect can be created against a single node of Isilon or CSI Driver can be installed/pointed to a particular node of the Isilon else basic authentication can be used by setting isiAuthType in `values.yaml` to 0 |
+| When an attempt is made to create more than one ReadOnly PVC from the same volume snapshot, the second and subsequent requests results in PVCs in state `Pending`, with a warning `another RO volume from this snapshot is already present`. This is because the driver allows only one RO volume from a specific snapshot at any point in time. This is to allow faster creation(with in few seconds) of a RO PVC from a volume snapshot irrespective of the size of volume snapshot. | Wait for the deletion of first RO PVC created from the same volume snapshot. |
```
Comment (nitesh3108) [inline content/docs/csidriver/troubleshooting/powerscale.md:20]: remove space --> (within few seconds)

Comment (bpjain2004) [reply content/docs/csidriver/troubleshooting/powerscale.md]: > remove space --> (within few seconds)

done

File: content/docs/csidriver/troubleshooting/powerscale.md [hunk 2]
```diff
@@ -16,4 +16,6 @@ Here are some installation failures that might be encountered and how to mitigat
...
+| While attaching a ReadOnly PVC from a volume snapshot to a pod, the mount operation will fail with error `mounting ... failed, reason given by server: No such file or directory`, if RO volume's access zone(non System access zone) on Isilon is configured with a dedicated service IP(which is same as `AzServiceIP` storage class parameter). This operation results in accessing the snapshot base directory(from /ifs) and results in overstepping the RO volume's access zone's base directory, which the OneFS doesn't allow. | Provide a service ip which belongs to RO volume's access zone which set the highest level /ifs as its zone base directory. |
```
Comment (nitesh3108) [inline content/docs/csidriver/troubleshooting/powerscale.md:21]: (from "/ifs")

Comment (bpjain2004) [reply content/docs/csidriver/troubleshooting/powerscale.md]: addressed this comment

**Review by prablr79 — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**